socialhome/streams/app/stores/streamStore.operations.js

```diff
@@ -0,0 +1,26 @@
+const stateOperations = {
+    receivedNewContent: "receivedNewContent",
+    newContentAck: "newContentAck",
+}
+
+const mutations = {
+    [stateOperations.receivedNewContent](state, newContentLengh) {
+        state.stream.hasNewContent = true
+        state.stream.newContentLengh += newContentLengh
+    },
+    [stateOperations.newContentAck](state) {
```

Comment:
New content acknowledgment. The user acknowledged new content is available by clicking the button and the state should be reset.